### PR TITLE
Update Auth0 Readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
Added link to Auth0 documentation

Type of change

- Enhancement

## What does this PR do?

Added link to Auth0 documentation

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/auth0/docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Auth0 Log Streams Integration
 
-Auth0 offers integrations that push log events via log streams to Elasticsearch. The Auth0 Log Streams integration package creates a HTTP listener that accepts incoming log events and ingests them into Elasticsearch. This allows you to search, observe and visualize the Auth0 log events through Elasticsearch.
+Auth0 offers integrations that push log events via log streams to Elasticsearch. The [Auth0 Log Streams](https://auth0.com/docs/customize/log-streams) integration package creates a HTTP listener that accepts incoming log events and ingests them into Elasticsearch. This allows you to search, observe and visualize the Auth0 log events through Elasticsearch.
```
Comment (r00tu53r) [inline packages/auth0/docs/README.md:3]: The log streams link mentioned above is a good place to read about what log streams do in Auth0. It is worth providing this link too. Perhaps rephrase this a bit to include both the links. 

- Auth0's integration to Elastic Security - https://marketplace.auth0.com/integrations/elastic-security

**Comment by efd6** [comment]
This needs `elastic-package build`.

File: packages/auth0/docs/README.md [hunk 2]
```diff
@@ -88,7 +88,6 @@ The Auth0 logs dataset provides events from Auth0 log stream. All Auth0 log even
 | destination.user.domain | Name of the directory the user is a member of. For example, an LDAP or Active Directory domain name. | keyword |
 | destination.user.id | Unique identifier of the user. | keyword |
 | destination.user.name | Short name or login of the user. | keyword |
-| destination.user.name.text | Multi-field of `destination.user.name`. | match_only_text |
```
Comment (r00tu53r) [inline packages/auth0/docs/README.md:91]: The update you've made doesn't need to change the fields documentation. Please check nothing else is changed for this package and re-build.

File: packages/auth0/docs/README.md (hunk 1 above)
Comment (ellis-elastic) [reply packages/auth0/docs/README.md:3]: Done

**Review by r00tu53r — APPROVED** [review]
LGTM. @ellis-elastic thank you for the contribution. You can merge after CI passes.

I noticed the CI was failing because the generated README.md (i.e. `docs/README.md`) was not committed/pushed so I pushed it.

**Pull request merged**